bin/save_singler_refs.R

```diff
@@ -12,15 +12,25 @@ option_list <- list(
     help = "name associated with celldex reference, e.g., HumanPrimaryCellAtlasData"
   ),
   make_option(
-    opt_str = c("--ref_file"),
+    opt_str = c("--ref_file_prefix"),
     type = "character",
-    help = "path to save reference file"
+    help = "prefix to use for naming saved reference file.
+      All files will be saved with the following name: <ref_file_prefix>_celldex_version.rds"
```

Comment:
Two suggestions - 

- seems like `celldex` is already part of `ref_file_prefix`
- adding `v` to match how you code it up below (or you can leave off the `v` all around, I don't think we need it? up to you)

```suggestion
      All files will be saved with the following name: <ref_file_prefix>_v<celldex_version>.rds"
```

Reply:
why is this so red!!!!!!!